tensorbay/opendataset/CCPD/loader.py

```diff
@@ -0,0 +1,260 @@
+                    444&547_368&549_364&517_440&515-0_0_22_10_26_29_24-128-7.jpg
+                    00221264367816-91_91-283&519_381&553- \
+                    375&551_280&552_285&514_380&513-0_0_7_26_17_33_29-95-9.jpg
+                    ...
+                ccpd_blur/
+                ccpd_challenge/
+                ccpd_db/
+                ccpd_fn/
+                ccpd_rotate/
+                ccpd_tilt/
+                ccpd_weather/
+                LICENSE
+                README.md
+                splits/
+                    ccpd_blur.txt
+                    ccpd_challenge.txt
+                    ccpd_db.txt
+                    ccpd_fn.txt
+                    ccpd_rotate.txt
+                    ccpd_tilt.txt
+                    test.txt
+                    train.txt
+                    val.txt
+
+    Arguments:
+        path: The root directory of the dataset.
+
+    Returns:
+        Loaded :class: `~tensorbay.dataset.dataset.Dataset` instance.
+
+    """
+    root_path = os.path.abspath(os.path.expanduser(path))
+
+    dataset = Dataset(DATASET_NAME_1)
+    dataset.load_catalog(os.path.join(os.path.dirname(__file__), "catalog.json"))
+
+    for segment_head, segment_tails in _CCPD_SEGMENTS.items():
+        for segment_tail in segment_tails:
+            segment_name = f"{segment_head}-{segment_tail}"
+            segment = dataset.create_segment(segment_name)
+            for image_path in _get_ccpd_image_path(root_path, segment_head, segment_tail):
+                data = Data(image_path)
+                data.label.polygon = _get_label(image_path) if segment_name != "other-np" else []
+                segment.append(data)
+    return dataset
+
+
+def CCPDGreen(path: str) -> Dataset:
+    """Dataloader of CCPDGreen open dataset.
+
+    .. _CCPD: https://github.com/detectRecog/CCPD
+
+    The file structure should be like::
+
+        <path>
+            CCPD2020/
+                ccpd_green/
+                    train/
+                    test/
+                    val/
+
+    Arguments:
+        path: The root directory of the dataset.
+
+    Returns:
+        Loaded :class: `~tensorbay.dataset.dataset.Dataset` instance.
+
+    """
+    root_path = os.path.abspath(os.path.expanduser(path))
+
+    dataset = Dataset(DATASET_NAME_2)
+    dataset.load_catalog(os.path.join(os.path.dirname(__file__), "catalog.json"))
+
+    for segment_tail in _CCPDGREEN_SEGMENTS:
+        segment = dataset.create_segment(f"{segment_tail}")
+        for image_path in glob(
+            os.path.join(root_path, "CCPD2020", "ccpd_green", f"{segment_tail}", "*.jpg")
+        ):
+            data = Data(image_path)
+            data.label.polygon = _get_label(image_path)
+            segment.append(data)
+    return dataset
+
+
+def _get_license_plate(license_index: str) -> str:
+    indexes = map(int, license_index.split("_"))
+    province_c = _PROVINCES[next(indexes)]
+    codes_c = _CODES[next(indexes)]
```

Comment:
What does `_c` stand for?

Reply:
character, just different from _PROVINCES and _CODES

Reply:
province and code are better